README.md

```diff
@@ -121,7 +121,7 @@ Validator                               | Description
 **isHexadecimal(str)**                  | check if the string is a hexadecimal number.
 **isHexColor(str)**                     | check if the string is a hexadecimal color.
 **isHSL(str)**                          | check if the string is an HSL (hue, saturation, lightness, optional alpha) color based on [CSS Colors Level 4 specification][CSS Colors Level 4 Specification].<br/><br/>Comma-separated format supported. Space-separated format supported with the exception of a few edge cases (ex: `hsl(200grad+.1%62%/1)`).
-**isIBAN(str)**                         | check if the string is an IBAN (International Bank Account Number).
+**isIBAN(str)**                         | check if the string is an IBAN (International Bank Account Number).<br/><br/>Also allows you to define a whitelist, when you only want to recieve IBAN codes from certain countries or a blacklist, removing some of them from the current list.
```

Comment:
Could you elaborate on how you can define this? Like we explain the `options` object in other validators as well. Also with the list of supported country codes

Reply:
@WikiRik I have updated and put which attributes it will receive and also the values that can be used in each of them.